R/appStatic.R

```diff
@@ -50,6 +50,28 @@ exportStaticApp <- function(result,
     cli::cli_inform(c("i" = "Provided directory does not exist, it will be created."))
     dir.create(path = directory, recursive = TRUE)
     cli::cli_inform(c("v" = "directory created: {.pkg {directory}}"))
+
+  } else if (file.exists(here::here(directory, "shiny"))) {
```

Comment:
use file.path instead of directory (as it won't necessarily be in the same project)